### PR TITLE
Add convert seconds to hours

### PR DESCRIPTION
**Pull request by elnexreal**
**Please describe the changes this PR makes and why it should be merged:**

I just modified the Convert seconds to day MOD and i used another syntax

Instead of showing <b>1D 24H 60M 60S</b> it shows <b>HH:MM:SS</b>

I think it should be added for showing

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [x] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by shinobihaxor** [comment]
Not sure if this should be a separate mod, maybe modify the current mod to a newer version where you can actually pick how you want it to be converted.

**Review by shinobihaxor — APPROVED** [review]

File: actions/convert_seconds_to_hours_MOD.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    let s = time
+    let m = Math.floor(s / 60)
+    s %= 60
+    let h = Math.floor(m / 60)
+    m %= 60
+    h %= 24
+
+    let result = `${h}:${m}:${s}`
+
+    if (result.toString() === 'Invalid Date') result = undefined
```
Comment (almeidx) [inline actions/convert_seconds_to_hours_MOD.js:55]: This will never happen

File: actions/convert_seconds_to_hours_MOD.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const time = this.evalMessage(data.time, cache)
+
+    if (isNaN(time)) return this.callNextAction(cache)
+
+    let s = time
+    let m = Math.floor(s / 60)
+    s %= 60
+    let h = Math.floor(m / 60)
+    m %= 60
+    h %= 24
```
Comment (almeidx) [inline actions/convert_seconds_to_hours_MOD.js:51]:
```suggestion
    const s = time % 60
    let m = Math.floor(time / 60)
    const h = Math.floor(m / 60) % 24
    m %= 60
```

File: actions/convert_seconds_to_hours_MOD.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  name: 'Convert Seconds To H:M:S',
+  section: 'Other Stuff',
+
+  subtitle (data) {
+    return `Convert ${data.time}`
+  },
+
+  variableStorage (data, varType) {
+    const type = parseInt(data.storage)
+    if (type !== varType) return
```
Comment (almeidx) [inline actions/convert_seconds_to_hours_MOD.js:11]:
```suggestion
    if (parseInt(data.storage) !== varType) return
```

File: actions/convert_seconds_to_hours_MOD.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  html (isEvent, data) {
+    return `
+<div style="float: left; width: 70%; padding-top: 8px;">
+  Seconds to Convert:
+  <input id="time" class="round" type="text" placeholder="e.g. 1522672056 or use Variables">
+</div>
+<div style="float: left; width: 35%; padding-top: 8px;">
+  Store Result In:<br>
+  <select id="storage" class="round" onchange="glob.variableChange(this, 'varNameContainer')">
+  ${data.variables[0]}
```
Comment (almeidx) [inline actions/convert_seconds_to_hours_MOD.js:26]:
```suggestion
    ${data.variables[0]}
```

File: actions/convert_seconds_to_hours_MOD.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  </select>
+</div>
+<div id="varNameContainer" style="float: right; display: none; width: 60%; padding-top: 8px;">
+  Variable Name:<br>
+  <input id="varName" class="round" type="text">
+</div><br><br>`
```
Comment (almeidx) [inline actions/convert_seconds_to_hours_MOD.js:32]:
```suggestion
</div>`
```

File: actions/convert_seconds_to_hours_MOD.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  },
+
+  init () {
+    const { glob, document } = this
+    glob.variableChange(document.getElementById('storage'), 'varNameContainer')
+  },
+
+  action (cache) {
+    const data = cache.actions[cache.index]
+    const time = this.evalMessage(data.time, cache)
```
Comment (almeidx) [inline actions/convert_seconds_to_hours_MOD.js:42]:
```suggestion
    const time = Number(this.evalMessage(data.time, cache))
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Comment by almeidx** [comment]
> Not sure if this should be a separate mod, maybe modify the current mod to a newer version where you can actually pick how you want it to be converted.

Are you going to be doing this?

**Comment by elnexreal** [comment]
I was able to do it, but i think it would be better if you make it to convert all times cuz im not good at JavaScript

**Pull request closed without merging**